lib/src/main/java/io/github/taz/java/instagram/api/IgClient.java

```diff
@@ -29,6 +32,7 @@ public final class IgClient {
     private final String username, password;
     private String authorization, cookies;
     private HttpClient httpClient = HttpClient.newHttpClient();
+    private static final Logger logger = LoggerFactory.getLogger(IgClient.class);
```

Comment:
move it at the top and leave a line gap between this and other fields, also ig the name should be `LOG` whats ur opinion on that?

Reply:
I prefer to use `logger`, though I am fine with `log` too.
I don't see why we should use uppercase.

Reply:
uppercase coz that javas convention for final vars?

Reply:
that's if they are public, afaik.

Reply:
can't find anything like that on google, and also when learning i didn't learnt about any exceptions here iirc